lib/less/tree/import.js

```diff
@@ -127,7 +127,7 @@ Import.prototype.eval = function (context) {
         }
     }
 
-    if (this.options.inline) {
+    if (this.options.inline && !this.path.currentFileInfo.reference) {
```

Comment:
I do not think you wrote something rude, unless you edited it away in the meantime. Suggestions like this are why I do not commit directly and send pull requests instead. It is possible for other people to review and suggest different/cleaner solution.

As I understand it, setting `isReferenced` wont help. That function is used to override "imported by reference therefore do not print" int ruleset/directive/selector/comment. Basically, nobody cares about whether `Anonymous` is stored in referenced file or not, it is printed

I can change the fix to produce Anonymous in eval and then do not print if Anonymous is in referenced file. I will do that later this week.